dotnet/src/Experimental/Process.Core/ProcessBuilder.cs

```diff
@@ -137,6 +138,26 @@ public ProcessEdgeBuilder OnExternalEvent(string eventId)
         return new ProcessEdgeBuilder(this, eventId);
```

Comment:
should there be a List or Hash set storing the external events that can trigger the process?

so then through something like `process.GetExternalInputTriggerEvents()` a list of the external events can be obtained